### PR TITLE
tools: support using tiup playground to bench region heartbeat

### LINKED ISSUE #5690: Support running heartbeat benchmark through tiup playground
## Enhancement Task
It will be more convenient for testing.

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Issue Number: Close #5690

### What is changed and how does it work?
Now, it can be run through `tiup playground nightly --kv=1 --kv.binpath=/path/to/pd-heartbeat-bench --kv.config=/path/to/config-template.toml --pd=1 --db=0 --host <your_host>`

```commit-message
```

### Check List

Tests

- Manual test

### Release note

```release-note
None
```

### CODE REVIEW

File: tools/pd-heartbeat-bench/main.go [hunk 1]
```diff
@@ -16,30 +16,37 @@ package main
...
 	"go.etcd.io/etcd/pkg/report"
 	"google.golang.org/grpc"
 )
 
 var (
 	pdAddr            = flag.String("pd", "127.0.0.1:2379", "pd address")
-	storeCount        = flag.Int("store", 40, "store count")
-	regionCount       = flag.Int("region", 1000000, "region count")
+	storeCount        = flag.Int("store", 100, "store count")
+	regionCount       = flag.Int("region", 1800000, "region count")
```
Comment (nolouch) [inline tools/pd-heartbeat-bench/main.go:40]: how to config it with `tiup`?

**Review by CabinfeverB — APPROVED** [review]

**Review by nolouch — COMMENTED** [review]
reset LGTM.
BTW,  How to deploy in different machine. benchmark may cost many resources.

File: tools/pd-heartbeat-bench/config/config.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+	Log      log.Config `toml:"log" json:"log"`
+	logger   *zap.Logger
+	logProps *log.ZapProperties
+
+	StoreCount        int     `toml:"store-count" json:"store-count"`
+	RegionCount       int     `toml:"region-count" json:"region-count"`
+	KeyLength         int     `toml:"key-length" json:"key-length"`
+	Replica           int     `toml:"replica" json:"replica"`
+	LeaderUpdateRatio float64 `toml:"leader-update-ratio" json:"leader-update-ratio"`
```
Comment (nolouch) [inline tools/pd-heartbeat-bench/config/config.go:42]: I think we still need command line arguments.

**Comment by lilinghai** [comment]
> reset LGTM. BTW, How to deploy in different machine. benchmark may cost many resources.
@nolouch first we start pd server with monitor by tiup playground in a machine, second we start the heartbeat workload in another machine.

**Review by lilinghai — APPROVED** [review]

File: tools/pd-heartbeat-bench/config/config.go (hunk 2 above)
Comment (rleungx) [reply tools/pd-heartbeat-bench/config/config.go:42]: It can be set through `--config`

**Review by nolouch — APPROVED** [review]

**Pull request merged**